deploy/crds/nmstate.io_nodenetworkconfigurationpolicies_crd.yaml

```diff
@@ -51,38 +51,106 @@ spec:
             of NodeNetworkConfigurationPolicy
           properties:
             enactments:
-              description: 'TODO: This is a temporary solution. This list will be
-                replaced by a dedicated NodeNetworkConfigurationEnactment object.'
-              items:
-                properties:
-                  conditions:
-                    items:
-                      properties:
-                        lastHearbeatTime:
-                          format: date-time
-                          type: string
-                        lastTransitionTime:
-                          format: date-time
-                          type: string
-                        message:
-                          type: string
-                        reason:
-                          type: string
-                        status:
-                          type: string
-                        type:
-                          type: string
-                      required:
-                      - status
-                      - type
-                      type: object
-                    type: array
-                  nodeName:
-                    type: string
-                required:
-                - nodeName
-                type: object
-              type: array
+              description: NodeNetworkConfigurationPolicyList contains a list of NodeNetworkConfigurationPolicy
```

Comment:
Are you nesting NNCE into NNCP? We only want to reference the NNCE from here.

Reply:
What do you mean ?

Reply:
But neither to include the whole object in it, right? This is a CRD for NNCP, with your change it contains list of enactments in their full format, why? We already have standalone NNCE in Kubernetes API, we don't want to see all NNCE when getting a single policy.

Reply:
Ahh wait didn't show it, let me remove it.